app/features-json/view_models/project_view_model.rb

```diff
@@ -3,7 +3,7 @@
 require_relative "../../shared/json_convertible"
 
 module FastlaneCI
-  # View model to expose the basic info about a project.
+  # View model to expose the detailed info about a project.
```

Comment:
@nakhbari can you confirm this is accurate? This and another class had the same class level description, and I wasn't 100% certain when which one is used

Reply:
yep this is good 👍